README.md

```diff
@@ -32,7 +32,7 @@ For more information on taps and the keg, see [keg-hub](https://github.com/simpl
 
     # This action will tag your tap's built image with this tag name.
     # 
-    # Default: 'develop'
+    # Default: tap_ref input value ( current branch name || develop )
```

Comment:
* This will allow it to default building images with the current branch name 
* This is makes it work exactly the same as the `keg <tap-name> package` command
* Which means when it runs for a PR, the image will have the branch name as the tag name